administrator/language/en-GB/com_scheduler.ini

```
@@ -0,0 +1,87 @@
+; Joomla! Projects
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; GNU General Public License version 2 or later; see LICENSE.txt
+
```

```suggestion
; Joomla! Project
; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
; License GNU General Public License version 2 or later; see LICENSE.txt
; Note : All ini files need to be saved as UTF-8

```

Please update all language files with the correct copyright header as shown above